plugins/node/instrumentation-runtime-node/src/types.ts

```diff
@@ -17,11 +17,5 @@ import type { InstrumentationConfig } from '@opentelemetry/instrumentation';
 
 export interface RuntimeNodeInstrumentationConfig
   extends InstrumentationConfig {
-  /**
-   * The approximate number of milliseconds for which to calculate event loop utilization averages.
-   * A larger value will result in more accurate averages at the expense of less granular data.
-   * Should be set to below the scrape interval of your metrics collector to avoid duplicated data points.
-   * @default 5000
-   */
-  eventLoopUtilizationMeasurementInterval?: number;
+  monitoringPrecision?: number;
```

Comment:
curious about this change, can you give more context why the rename?

Reply:
Because right now i use it for every metric, not only for eventLoopUtilization. You think I needed to add parameters for every new metric group ?

Reply:
No need